package.json

```diff
@@ -38,16 +38,17 @@
     "typescript": "~4.6.3"
   },
   "dependencies": {
+    "@aws-lambda-powertools/logger": "^0.7.1",
     "@aws-sdk/client-s3": "^3.58.0",
     "@aws-sdk/client-sfn": "^3.58.0",
     "@aws-sdk/client-sqs": "^3.58.0",
     "@middy/core": "^2.5.7",
     "@middy/http-json-body-parser": "^2.5.7",
     "@middy/validator": "^2.5.7",
     "aws-cdk-lib": "2.17.0",
+    "cdk-iam-floyd": "0.345.0",
     "cdk-nag": "^2.12.5",
     "constructs": "^10.0.98",
-    "cdk-iam-floyd": "0.345.0",
```

Comment:
Should we bother to remove these changes since it's now in develop?

Reply:
I think this is an artifact of the merge conflict on the file

Reply:
I'll already rebased, so it should remove that.

Reply:
Yeah, appears to be an artifact like Kyle mentioned.